transitions/StereoViewer.glsl

```diff
@@ -0,0 +1,210 @@
+// Tunable parameters
+// How much to zoom (out) for the effect ~ 0.5 - 1.0
+uniform float zoom; // = 0.88
+// Corner radius as a fraction of the image height
+uniform float corner_radius;  // = 0.22
+
+// author: Ted Schundler
+// license: BSD 2 Clause
```

Comment:
'BSD 2 Clause' not found in supported licenses: MIT, EPL-1.0, CDDL-1.0, MPL-2.0, LGPL, GPLv2, GPLv3, Apache-2.0